src/gwkokab/models/utils/_lazyjointdistribution.py

```
@@ -0,0 +1,269 @@
+                    x,
+                    event_slice[0],
+                    event_slice[1] - event_slice[0],
+                    axis=-1,
+                )
+            if mask is None:
+                mask = constraint.check(x_slice)
+            else:
+                mask = jnp.logical_and(mask, constraint.check(x_slice))
+        return mask  # type: ignore
+
+    def tree_flatten(self):
```

Ruff (0.14.2)

Missing return type annotation for private function `tree_flatten`

(ANN202)